lib/Cake/Utility/CakeTime.php

```diff
@@ -1176,4 +1182,29 @@ protected static function _strftime($format, $date) {
 		return $format;
 	}
 
+/**
+ * Multibyte wrapper for strftime.
+ *
+ * Adjusts the timezone when necessary before formatting the time.
+ *
+ * @param string $format Format string.
+ * @param int $timestamp Timestamp to format.
+ * @param int|string|DateTime $date Timestamp, strtotime() valid string or DateTime object.
+ * @param string|DateTimeZone $timezone Timezone string or DateTimeZone object.
+ * @return string Formatted date string with correct encoding.
+ */
+	private static function __strftimeWithTimezone($format, $timestamp, $date, $timezone) {
```

Comment:
`protected`?

Reply:
:+1: to protected we generally don't use many private methods.